arm-web/2016-09-01/swagger/AppServiceEnvironments.json

```diff
@@ -4197,6 +4414,67 @@
     "WorkerPool": {
```

Comment:
please check properties sharing described above can be applied for WorkerPool and WorkerPoolResource